### ISSUE TITLE
RFM - rsync

### ISSUE THREAD

**Issue by mcandre** [chocolatey/chocolatey-package-requests#366]
The rsync package is broken, at least for x86 users, as a result of the rsync installation media being hosted, in quite a fragile manner, first in https://www.itefix.net/dl/cwRsync_5.5.0_x86_Free.zip (a dead website), and then indirectly via the Wayback Machine https://web.archive.org/web/20160613105413/https://www.itefix.net/dl/cwRsync_5.5.0_x86_Free.zip , whose download link is currently inoperable.

In short, rsync (esp. x86 edition) requires a more stable file host, such as GitHub, SourceForge, BitBucket, GitLab, or MediaFire.

**Comment by pauby** [comment]
Package is at https://chocolatey.org/packages/rsync

**Comment by pauby** [comment]
No response. Closing.

**Comment by pauby** [comment]
Opened for other maintainers.

**Comment by mcandre** [comment]
The updated package seems to work for me on both 32 and 64 bit.
